### ISSUE TITLE
NVDA doesn't read anything on iTunes

### ISSUE THREAD

**Issue by Choominator** [nvaccess/nvda#8744]
Today I woke up and NVDA was no longer reading anything on iTunes.  After investigating I figured that iTunes was updated to version 12.9.0.167 from the Microsoft Store overnight, so suspecting it might be an issue with iTunes I tried Narrator which worked just fine.

I'm running Microsoft Windows 10 Home version 10.0.17134 Build 17134 with NVDA 2018.2.1, and after analyzing the [log](https://github.com/nvaccess/nvda/files/2379550/nvda.log) I found that there were exceptions every time I tried to interact with iTunes.

**Comment by leonardder** [comment]
It looks like ITunes switched to UIA.

A quick workaround might be force disabling UIA for now on the app module level, or update the appModule to be compatible with those.

Marking this as a p2 regression, since ITunes is a widely used product and these issues seem pretty serious. Furthermore, they are caused by the NVDA appModule

**Comment by derekriemer** [comment]
a fix is in for this in 2018.3
